book/website/ethical-research/law-policy.md

```diff
@@ -0,0 +1,263 @@
+Some campaigning groups are international and work on lots of different human rights issues. 
+Some work only on a small number of issues, or in a small area of the world.
+
+### How do human rights apply to data science and research ethics?
+
+Human rights are defined in law. 
+This means in many ways they are less malleable than ethics.
+The letter of human rights, and the ways that they are implemented, are agreed internationally,
+A violation of human rights is a violation wherever it happens. 
+This is not necessarily true of codes of ethics. 
+An action that one group of people sees as ethical, may be seen as unethical by others. 
+There is a wider shared understanding of what human rights means across different countries, cultures and groups of people.
+There are accountability and enforcement mechanisms.
+But because they are a legal framework, they are not as easily accessible, and hard to apply to the daily work of data science and research.
+
+Human rights treaties are signed by countries: that means that ultimately, it is countries that are held responsible for violations of human rights. 
+However, that does not mean that researchers and data scientists are exempt from thinking about the human rights impact of their work!
+
+The UN has produced specific guidance for companies on how to ensure that they respect human rights: the [UN Guiding Principles on Business and Human Rights](https://www.business-humanrights.org/en/big-issues/un-guiding-principles-on-business-human-rights/).
+Companies should comply with these Guiding Principles.
+
+NGOs and other campaigning organisations also produce guidance on specific areas:
+- [The Toronto Declaration: Protecting the rights to equality and non-discrimination in machine learning systems](https://www.accessnow.org/the-toronto-declaration-protecting-the-rights-to-equality-and-non-discrimination-in-machine-learning-systems/)(_prepared by Amnesty International and Access Now and it has been endorsed by Human Rights Watch and Wikimedia Foundation_)
+
+Making sure that research complies with human rights is not just about ensuring that it doesn't harm.
+It can also be about doing work that is actively pro-rights. 
+
+Researchers and data scientists should, in general ensure that they:
+- **respect** human rights: this means not doing work that itself abuses human rights 
+- **protect** human rights: this means trying to ensure that their work cannot be used by others to abuse human rights
+- **fulfil** human rights: this means doing work that actively helps other people be able to enjoy all their human rights
+
+## Checklist
+- [ ] Do I know what laws apply to my work? 
+- [ ] Do I know what policies apply to my work?
+- [ ] Have I read relevant policies on ethical research?
+- [ ] Does my work respect and protect human rights?
+- [ ] Does my work fulfil human rights?
```

Comment:
Add: Have I engaged with the organisation's Governance/Policy teams? (Make it a more proactive approach)

Reply:
added!